packages/data-context/src/actions/TestActions.ts

```diff
@@ -0,0 +1,129 @@
+import path from 'path'
+import globby from 'globby'
+import assert from 'assert'
+import fs from 'fs-extra'
+import disparity from 'disparity'
+import tempDir from 'temp-dir'
+
+// Also defined in lib/fixtures, but we do not want to
+// import from system-tests into code we ship in production.
+const cyTmpDir = path.join(tempDir, 'cy-projects')
```

Comment:
why are we shipping test code in production? this seems very wrong

Reply:
Yeah not super ideal - I will move this into a `cy.task`.